src/smartAuthorizationHelper.ts

```diff
@@ -112,6 +113,24 @@ export function hasReferenceToResource(
     );
 }
 
+export function hasReferenceToPatientOrgs(
+    patientOrgs: FhirResource[],
+    sourceResource: any,
+    apiUrl: string,
+    fhirVersion: FhirVersion,
+): boolean {
+    let isPresent: boolean = false;
```

Comment:
Is the intent of this function to return true on the the first org reference found?  If so, there's the much more concise some method: `patientOrgs.some(org => hasReferenceToResource(org, sourceResource, apiUrl, fhirVersion))`.